service/relayer/Dockerfile

```
@@ -1,28 +1,51 @@
+# The RedStone bridge build script runs npm during cargo build.
+RUN node --version && npm --version
+RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/usr/local/cargo/git/db \
     --mount=type=cache,target=/usr/local/rustup \
     cargo build --package templar-relayer --profile release
 
-# Runtime image
-FROM debian:12-slim
+FROM debian:bookworm-slim AS production
 
 RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)